packages/fields/src/fields/parent/parent-view.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { store as coreStore } from '@wordpress/core-data';
+
+/**
+ * Internal dependencies
+ */
+import type { BasePost } from '../../types';
+import type { DataViewRenderFieldProps } from '@wordpress/dataviews';
+import { getTitleWithFallbackName } from './utils';
+import { __ } from '@wordpress/i18n';
+
+export const ParentView = ( {
+	item,
+}: DataViewRenderFieldProps< BasePost > ) => {
+	const parent = useSelect(
+		( select ) => {
+			const { getEntityRecord } = select( coreStore );
+			return item?.parent
+				? getEntityRecord( 'postType', item.type, item.parent )
+				: null;
+		},
+		[ item.parent, item.type ]
+	);
+
+	if ( parent ) {
+		return <>{ getTitleWithFallbackName( parent ) }</>;
+	}
+
+	return <>{ __( 'None' ) }</>;
```

Comment:
This works for me: we could land as it is and then gather feedback from folks. Though I wonder if there's a better default for this because `None` could be an actual title, so it may be confusing. Perhaps `—` like wp-admin uses in some places?

Reply:
I would proceed in this way. 

`None` is used in the Document Inspector panel:

![image](https://github.com/user-attachments/assets/eb7d186b-cbdc-499e-be14-7b9f15bf2ab2)